src/test/scala/uk/gov/nationalarchives/LambdaTest.scala

```diff
@@ -0,0 +1,77 @@
+package uk.gov.nationalarchives
+
+import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyResponseEvent
+import org.mockito.ArgumentMatchers.any
+import org.mockito.Mockito.when
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+import org.scalatestplus.mockito.MockitoSugar.mock
+import software.amazon.awssdk.services.dynamodb.DynamoDbClient
+import software.amazon.awssdk.services.dynamodb.model._
+import uk.gov.nationalarchives.referencegenerator.Lambda
+
+import scala.jdk.CollectionConverters.MapHasAsJava
+
+class LambdaTest extends AnyFlatSpec with Matchers {
+
+  "The Lambda class" should "successfully process a request with the correct number of references" in {
+    val mockDDB = mock[DynamoDbClient]
+    val input = Lambda.Input(numberOfReferences = 3)
+    val lambda = new Lambda()
+    val getItemMap = Map("pieceCounter" -> AttributeValue.builder().n("10").build()).asJava
+    val getItemResponse = GetItemResponse.builder().item(getItemMap).build()
+
+    when(mockDDB.getItem(any[GetItemRequest])).thenReturn(getItemResponse)
+
+    val updateItemMap = Map("pieceCounter" -> AttributeValue.builder().n("12").build()).asJava
+    val updateItemResponse = UpdateItemResponse.builder().attributes(updateItemMap).build()
+
+    when(mockDDB.updateItem(any[UpdateItemRequest])).thenReturn(updateItemResponse)
+
+    val actual: APIGatewayProxyResponseEvent = lambda.process(input, mockDDB)
+    val expected: APIGatewayProxyResponseEvent = new APIGatewayProxyResponseEvent()
+      .withStatusCode(200)
+      .withBody("""["N","P","Q"]""")
+    actual shouldBe expected
+  }
+
+  it should "return an exception message if getItem fails" in {
```

Comment:
Could you keep the same style as above and have the test written as: `"The Lambda class" should ...`